rllab/envs/bullet_env.py

```diff
@@ -0,0 +1,138 @@
+import gym
```

Comment:
If the code is all the same, why not just use GymEnv? Or inherit from it if there are small differences?

BulletEnvs use the Gym interface, so all the wrapping code should be the same. Are there differences in the initialization or plotting perhaps?

Reply:
No, the only difference was in registering the new environment which `pybullet` doesn't do.
Because of that, `env.spec` is `None`.

Reply:
Then definitely don't copy code, just inherit.